images/Dockerfile.ubuntu-20-lts-arm-rpi

```diff
@@ -0,0 +1,61 @@
+ARG BASE_IMAGE=ubuntu:20.04
+
+FROM $BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:oibaf/test && \
+    apt-get update && \
+    apt-get install -y \
+    conntrack \
+    console-data \
+    coreutils \
+    curl \
+    debianutils \
+    dosfstools \
+    dracut-network tar \
```

Comment:
I just noticed this is true in the other debian/ubuntu ones as well, but is there a reason to have two packages on a single line?

Reply:
fixed